staging/src/k8s.io/client-go/transport/cache.go

```diff
@@ -109,6 +110,7 @@ func tlsConfigKey(c *Config) (tlsCacheKey, error) {
 		caData:     string(c.TLS.CAData),
 		certData:   string(c.TLS.CertData),
 		keyData:    string(c.TLS.KeyData),
+		getCert:    fmt.Sprint(c.TLS.GetCert),
```

Comment:
what does this print as? I expected %p like we use for c.Dial

Reply:
also, if you're changing this, update TestTLSConfigKey to test for uniqueness

Reply:
`fmt.Sprint` on functions is same as `%p`. Changed for consistency.
Updates the test too.